core/common/src/test/java/alluxio/collections/IndexedSetTest.java

```diff
@@ -53,11 +53,21 @@ public long longValue() {
   @Before
   public void before() {
     mIntIndex = new IndexedSet.FieldIndex<Pair>() {
+      @Override
+      public Boolean isUnique() {
```

Comment:
optional: it might make sense to create `UniqueFieldIndex<T>` and `NonUniqueFieldIndex<T>` abstract classes that already implement isUnique to reduce the boilerplate for creating a new `FieldIndex`